### PR TITLE
Add Lda recipe

### PR DESCRIPTION
**Pull request by mwilson8**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/lda/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 50b91f80aff1b138b8b78f48d581c0e685e7adb393da9e45905773d5f4cc8bf5
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py>=37]
```
Comment (mwilson8) [inline recipes/lda/meta.yaml]: If building without skipping `py>=37` then an error like this occurs: https://github.com/kevlar-dev/kevlar/issues/282

If there is a way to fix and allow building on py>=37, please let me know

**Comment by mwilson8** [comment]
@rluria14 maintain this with me?

File: recipes/lda/meta.yaml (hunk 1 above)
Comment (marcelotrevisani) [reply recipes/lda/meta.yaml]: don't you need ``cython`` for this project?

File: recipes/lda/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  host:
+    - pip
+    - python
+    - numpy
+    - pbr
+    - setuptools
+    
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/lda/meta.yaml]: I think you also need ``numpy`` here, am I right?
```suggestion
    - python
    - numpy
```

File: recipes/lda/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+test:
+  imports:
+    - lda
+    - lda.datasets
```
Comment (marcelotrevisani) [inline recipes/lda/meta.yaml]:
```suggestion
    - lda.datasets
    - lda.utils
```

**Comment by mwilson8** [comment]
@marcelotrevisani changes applied

File: recipes/lda/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  host:
+    - pip
+    - python
+    - numpy
+    - pbr
+    - setuptools
+    
+  run:
+    - python
+    - numpy
```
[the remark this answers is not in the document]
Comment (mwilson8) [reply recipes/lda/meta.yaml]: Thanks; it does seem to fit both of the "dead giveaways". Changes made in latest commit

**Pull request merged**